blog.html

```diff
@@ -22,6 +22,286 @@
+                        This improvement simplifies joins, reduces storage overhead, and provides a more intuitive
+                        developer experience.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/TZCMEw">KIP-1112: allow custom processor wrapping</a>
+                        <br>
+                        KIP-1112 simplifies applying cross-cutting logic in Kafka Streams by introducing the
+                        ProcessorWrapper interface, enabling seamless injection of custom logic around Processor API
+                        and DSL processors. This eliminates previous redundancy and reduces the maintenance
+                        overhead caused by manually integrating logic into each processor individually.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/LQ6TEg">KIP-1065: Add "retry" return-option to ProductionExceptionHandler</a>
+                        <br>
+                        KIP-1065 addresses persistent errors in Kafka Streams by allowing users to break retry loops
+                        via a new "RETRY" option in the ProductionExceptionHandler. This enhancement provides
+                        customizable error handling, enabling retries, graceful failures, or dropping problematic records
+                        to continue processing efficiently.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/IgstEw">KIP-1091: Improved Kafka Streams operator metrics</a>
+                        <br>
+                        Kafka Streams exposes its metrics with KIP-1076. The KIP especially adds a state metric for
+                        each StreamThread and the client instance itself. It also provides detailed visibility into the
+                        application state.
+                    </li>
+                </ul>
+
+                <h3>Kafka Connect</h3>
+
+                <ul>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/rZWzDw">KIP-970: Deprecate and remove Connect's redundant task configurations endpoint</a>
+                        <br>
+                        The endpoint <code>GET /connectors/{connector}/tasks-config</code> added in KIP-661 was deprecated in
+                        Apache Kafka 3.7 and is finally removed in 4.0. The endpoint
+                        </code>GET /connectors/{connector}/tasks</code> must be used as a replacement.
+                    </li>
+                    <li>
+                        <a href="https://cwiki.apache.org/confluence/x/jA3OEg">KIP-1074: Allow the replication of user internal topics</a>
+                        <br>
+                        Currently, Mirror Maker 2 automatically excludes topics ending in ".internal" or "-internal", 
```

Comment:
It's a bit strange to say `Currently` to describe the previous behavior. Should we say `Previously` and use the past tense in the first sentence or rephrase it?
Same below.